integrationTests/testProcessorNode.go

```diff
@@ -1881,10 +1881,11 @@ func (tpn *TestProcessorNode) initBlockProcessor(stateCheckpointModulus uint) {
 				return nil
 			},
 		},
-		BlockTracker:       tpn.BlockTracker,
-		BlockSizeThrottler: TestBlockSizeThrottler,
-		HistoryRepository:  tpn.HistoryRepository,
-		EpochNotifier:      tpn.EpochNotifier,
+		BlockTracker:             tpn.BlockTracker,
+		BlockSizeThrottler:       TestBlockSizeThrottler,
+		HistoryRepository:        tpn.HistoryRepository,
+		EpochNotifier:            tpn.EpochNotifier,
+		TransactionsLogProcessor: &mock.TxLogsProcessorStub{},
```

Comment:
Wondering if, for debugging purposes only, we should have a PrintTxLogProcessor implementation that could have dumped the logs on the default logger.

Reply:
done. added an implementation of the `TransactionLogProcessor` in the integration tests that will log all the events.